text/0000-dynamic-attributes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Passing Dynamic Attributes/Properties
+status: DRAFTED
+created_at: 2021-07-05
+updated_at: YYYY-MM-DD
+champion: Mohammed Abdul Sattar (abdulsattar)
+pr: https://github.com/salesforce/lwc-rfcs/pull/52
+---
+
+# Passing Dynamic Attributes/Properties
+
+## Summary
+This proposal adds mechanism to pass a dynamic attributes or properties bag to elements in
+an LWC template.
+
+## Example
+```js
+// x/myComponent.js
+export default class MyComponent extends LightningElement {
+    childProps = { 
+        name: 'My Child',
+        style: 'width: 50px',
+        class: 'my-class',
+        onclick: this.handleClick,
+    };
+    
+    handleClick() {
+        console.log('Clicked')
+    }
+}
+```
+```html
+// x/myComponent.html
+<template>
+    <x-child lwc:spread={childProps}></x-child>
```

Comment:
One of the requirements I have with this proposal, it that it should work on both standard and custom elements.

Reply:
That's the same expectation I have as well. I'll add an example for standard element to make it clear.